src/universal-app/tsconfig-prerender.json

```diff
@@ -5,7 +5,7 @@
     "stripInternal": false,
     "experimentalDecorators": true,
     // TODO(paul): Remove once Angular has been upgraded and supports noUnusedParameters in AOT.
```

Comment:
Should this comment be removed as it was in the other json files?

Reply:
Good point. Done